packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -268,7 +290,14 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 */
 	public ngOnInit(): void {
 		this.logger.debug(componentName + ": component initialized");
-		this.displayedColumns = [];
+
+		// Emit event when selection changes
+		if (this.selection.changed) {
+			this._selectionSub = this.selection.changed.subscribe((change: SelectionChange<object>) => {
+				const selected: object[] = change.source.selected;
+				this.selectChanged.emit(selected);
```

Comment:
Now that we use the `SelectionModel` to handle the selection, the array we emit is an array of objects (rows) while in the old Stark it was an array of ids. So could you add this API change in the Confluence page please about Stark 9 vs Stark 10?